### ISSUE TITLE
Keep Cookies in the HTTP Request

### ISSUE THREAD

**Issue by UltimateCodeLove** [negrutiu/nsis-nscurl#40]
I am trying to download a direct link and it requires cookies in the http request. Inetc has cookies, NSCurl removes cookies? How to keep it?

**Comment by UltimateCodeLove** [comment]
I'm getting https://login.microsoftonline.com/cookiesdisabled when trying to download the direct file. It works in Inetc

**Comment by UltimateCodeLove** [comment]
For example this works in curl: curl -o test.zip -L -A 'Mozilla' -c cookies.txt -b cookies.txt 'https://example.com/test.zip'

**Comment by negrutiu** [comment]
Indeed, the cookie engine is currently disabled.. Thanks for reporting!
I've made some changes that should fix the issue.
A temporary build is here: https://github.com/negrutiu/nsis-nscurl/actions/runs/12569688203/artifacts/2376122762
Could you give it a try and let me know how it goes?

**Comment by negrutiu** [comment]
Released the experimental changes. Feel free to reopen the ticket if something goes wrong
